### ISSUE TITLE
Enable logging on LB

### ISSUE THREAD

**Issue by jamilbk** [CloudFire-LLC/cloudfire#5772]
Why:

We need to be able to block certain bots or abusive IP ranges. Found the source of the traffic.

**Comment by AndrewDryga** [comment]
@jamilbk do we still want to do this? Logging can get pretty expensive if we start logging everything, plus it will only apply for the portal traffic anyways.

**Comment by jamilbk** [comment]
Yeah, we need it at some level. I haven't had a chance to research how much that would be or how to minimize costs, but for example we the ability to know who may be abusing the service.
